base_tier_validation_correction/tests/test_tier_validation.py

```diff
@@ -0,0 +1,135 @@
+# Copyright 2018 ForgeFlow S.L.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+from datetime import timedelta
+
+from odoo import fields
+from odoo.exceptions import ValidationError
+from odoo.tests.common import tagged
+
+from odoo.addons.base_tier_validation.tests.common import CommonTierValidation
+
+
+@tagged("post_install", "-at_install")
+class TierTierValidation(CommonTierValidation):
+    def test_01_tier_correction(self):
+        """With the document in validation,
+        - User click on Change Reivewer to creat new correction
+        - Change the reviewer to user 2, test that user 2 need_validation
+        - Revert to user 1, test that now user 1 need validation
+        - Click on Change Review link from the document again, then view the corrections
+        """
+        # User 2, request validation
+        doc_user2 = self.test_record.with_user(self.test_user_2.id)
+        doc_user2.request_validation()
+        self.assertFalse(doc_user2.can_review)
+        # User 1, is the reviewer as specified in the tier.definition
+        doc_user1 = self.test_record.with_user(self.test_user_1.id)
+        doc_user1._invalidate_cache()
```

Comment:
```suggestion
        doc_user1.invalidate_recordset()
```
invalidate_cache() is deprecated.
Please change this here, among with the other places it occurs in this file